support-frontend/assets/helpers/storage/abandonedBasketCookies.ts

```diff
@@ -0,0 +1,21 @@
+import * as cookie from 'helpers/storage/cookie';
+import { useEffect } from 'react';
+
+export function setAbandonedBasketCookie(
```

Comment:
Since this is using a hook it should start with the word `use` ([rules of hooks](https://react.dev/reference/rules/rules-of-hooks))

Reply:
Done